deepfence_agent/Makefile

```
@@ -38,6 +40,17 @@ gocode:
 
 install:
 
+agent:
+	$(CD) tools/apache/scope && \
+		go mod tidy -v && \
+		go mod vendor && \
+		env GOGC=off \
+		go build \
+		-ldflags "-X main.version=$(shell git rev-parse --short HEAD)  -X github.com/weaveworks/scope/probe/host.agentCommitID=$(shell git rev-parse --short HEAD) -X github.com/weaveworks/scope/probe/host.agentBuildTime=$(shell date +"%s%d%m%y") -s -w" \
+		-tags 'netgo osusergo unsafe' \
+		-o docker/deepfence_exe \
+		./prog
```

```suggestion
	($(CD) tools/apache/scope && \
		go mod tidy -v && \
		go mod vendor && \
		env GOGC=off \
		go build \
		-ldflags "-X main.version=$(shell git rev-parse --short HEAD)  -X github.com/weaveworks/scope/probe/host.agentCommitID=$(shell git rev-parse --short HEAD) -X github.com/weaveworks/scope/probe/host.agentBuildTime=$(shell date +"%s%d%m%y") -s -w" \
		-tags 'netgo osusergo unsafe' \
		-o docker/deepfence_exe \
		./prog)
```
This will prevent the shell from changing its location after doing `make agent`